src/foundry/index.d.ts

```diff
@@ -123,6 +124,9 @@ import './foundry.js/pixi/containers/canvasLayers/sightLayer';
 
 import './foundry.js/pixi/containers/canvasLayers/placeablesLayers/drawingsLayer';
 import './foundry.js/pixi/containers/canvasLayers/placeablesLayers/lightingLayer';
+import './foundry.js/pixi/containers/canvasLayers/placeablesLayers/mapLayer';
+import './foundry.js/pixi/containers/canvasLayers/placeablesLayers/mapLayers/backgroundLayer';
```

Comment:
Could you please create a structure if `index.js` files so that here, only `'./foundry.js/pixi/containers/canvasLayers'` needs to be imported? Ideally, we would only import `foundry.js` and `common/module.mjs` here at some point, but that's out of scope for this PR.